pkg/deployments/tasks/20220906-batch-relayer-v4/readme.md

```diff
@@ -0,0 +1,10 @@
+# 2022-09-06 - Batch Relayer V4
+
+Deployment of the fourth `BalancerRelayer` using `BatchRelayerLibrary`, for combining multiple operations (swaps, joins, etc.) in a single transaction.
+
+This new version adds support for Liquidity Gauge interactions, including staking, unstaking, minting, etc.
```

Comment:
This is what v3 does :stuck_out_tongue: 

I'd replace this with something like 'this new version provides quality of life improvements, like read only chained references and ways to read them'.